src/mmrelay/matrix/room_mapping.py

```
@@ -0,0 +1,365 @@
+from typing import (
+    Any,
+    Awaitable,
+    Callable,
+    Dict,
+    Generator,
+    Optional,
+    Tuple,
+)
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Awaitable`, `Callable`, `Generator`

Import from `collections.abc`

(UP035)